doc/getting-started/quick_start.ipynb

```diff
@@ -1257,21 +1354,31 @@
       "Ave neighs/atom = 4\n",
       "Neighbor list builds = 500\n",
       "Dangerous builds not checked\n",
-      "Total wall time: 0:00:13\n"
+      "Total wall time: 0:00:45\n"
      ]
     }
    ],
    "source": [
-    "! cd ../02.lmp && cp ../01.train/graph.pb ./ && lmp -i in.lammps"
+    "! cd ./DeePMD-kit_Tutorial/02.lmp && lmp -i in.lammps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3c1f883a-a768-4de1-8681-effd90643fee",
+   "metadata": {},
+   "source": [
+    "## References <a id ='references'></a>\n",
+    "\n",
+    "1. https://docs.deepmodeling.com/projects/deepmd/en/master/index.html\n",
```

Comment:
We don't need the reference considering it has already been in the documentation